tm2/pkg/bft/config/config.go

```diff
@@ -31,14 +31,14 @@ var (
 )
 
 const (
-	levelDBName  = "goleveldb"
-	clevelDBName = "cleveldb"
-	boltDBName   = "boltdb"
+	LevelDBName  = "goleveldb"
+	ClevelDBName = "cleveldb"
+	BoltDBName   = "boltdb"
 )
 
 const (
-	localABCI  = "local"
-	socketABCI = "socket"
```

Comment:
We don't define these values anywhere apart from comments (assuming people will set them correctly), even though we export all other config values (ex. db names). Our individual config validation methods are public, and the values we validate in the config are public as well

I've exported them out so they are visible to users who want to use the `config` package, and because I actually utilize this value inside a test:

https://github.com/zivkovicmilos/gno/blob/fb2b385c48cb706546f3d0d4599fc953ea400dee/gno.land/cmd/gnoland/config_edit_test.go#L240